src/tab.rs

```diff
@@ -1358,6 +1375,35 @@ fn folder_name<P: AsRef<Path>>(path: P) -> (String, bool) {
     (name, found_home)
 }
 
+// parse .hidden file and return files path
+fn parse_hidden_file(path: &PathBuf) -> Vec<PathBuf> {
+    let parent_dir = match path.parent() {
+        Some(dir) => dir,
+        None => return Vec::new(),
+    };
+
+    let file = match File::open(path) {
+        Ok(f) => f,
+        Err(_) => return Vec::new(),
+    };
+
+    let reader = BufReader::new(file);
+    let mut paths: Vec<PathBuf> = Vec::new();
+
+    for line in reader.lines() {
+        if let Ok(line) = line {
+            if !line.is_empty() {
+                let file_path = parent_dir.join(&line.trim());
```

Comment:
Converting to a full path is probably not necessary. You can store line.trim() in the Vec, and make it a Vec<String>, then test the item's name instead of path.